server/storage/storage.go

```diff
@@ -30,6 +29,10 @@ const (
 	ApiEntityName = "Api"
 	// VersionEntityName is the storage entity name for API version resources.
 	VersionEntityName = "Version"
+	// SpecEntityName is the storage entity name for API spec resources.
```

Comment:
let's talk offline about the decision to move these names from the models package to the storage package - it seems to be part of sanitizing the models layer to be independent of the storage details, but there are other things in the models package that are storage-dependent (the datastore and gorm annotations on struct fields), so this is a fuzzy boundary. 

I originally added the storage package as a way to abstract the interface to different storage backends (datastore, sql) and used the models package to describe most entity-specific details. A few things broke that (deletions seem like the biggest), but it was still fairly thin and minimal, making it relatively easy to consider adding other backends if needed.

Reply:
since this is a multi-pr change that's already in progress, I don't think we should stop it, but it's an internal detail that we may want to revisit.

Reply:
I agree, let's revisit this. My reasoning for the move was actually that these entity names solely used as arguments to storage.Client methods, so it made sense to locate them in the same package as the Client interface.